### DISCUSSION TITLE
Add Unity Compatibility to This Library

### DISCUSSION THREAD

**Discussion by daperez03** [vkhorikov/CSharpFunctionalExtensions#585] [Ideas]
Many developers use Unity for game development, but it’s often a challenge due to the lack of compatible libraries. Making this library Unity-compatible would greatly benefit the community and enhance its usability in game development projects.

**Comment by vkhorikov** [comment]
I guess it's just a matter of adding another target to the library? In any case, feel free to submit a PR, all enhancements are welcome.
